source/DailyOverview/DailyJS.js

```diff
@@ -33,34 +33,34 @@ let currentDay;
 
 window.addEventListener('load', () => {
     //gets the session, if the user isn't logged in, sends them to login page
-    let session = window.sessionStorage;
-    console.log('here is storage session', session);
-    if (session.getItem('loggedIn') !== 'true') {
-        window.location.href = '../Login/Login.html';
-        //might need this to create uness entires?
-        return;
-    } else {
-        let dbPromise = initDB();
-        dbPromise.onsuccess = function (e) {
-            console.log('database connected');
-            setDB(e.target.result);
-            // get the day and also the monthly and yearly goals
-            requestDay();
-            fetchMonthGoals();
-            fetchYearGoals();
-            let req = getSettings();
-            req.onsuccess = function (e) {
-                // gets the user settings, and in particular theme to set the style/background to the theme of user
-                let settingObj = e.target.result;
-                console.log('setting theme');
-                document.documentElement.style.setProperty(
-                    '--bg-color',
-                    settingObj.theme
-                );
-            };
+    // let session = window.sessionStorage;
```

Comment:
Is there a check that Firebase does to see if the user is logged in? I think we originally put this block of code here to prevent users from accessing the index page directly by merely visiting the HTML page by typing the URL directly

Reply:
The old getUserId() does that, so it would be called by every window.onload of every page to check if the user exists and gets their user id otherwise it redirects the user to the login page.